src/script/components/TitleBar/TitleBar.tsx

```diff
@@ -71,6 +72,7 @@ export const TitleBar: React.FC<TitleBarProps> = ({
   userState = container.resolve(UserState),
   callState = container.resolve(CallState),
   teamState = container.resolve(TeamState),
+  isReadOnlyConversation = false,
```

Comment:
To remove 😄 (I know it was for testing)

Reply:
Nevermind, i've realised it's a default prop value, thought it was overwritten in component's body, all good 👌